sopel/modules/instagram.py

```diff
@@ -9,8 +9,6 @@
 from __future__ import unicode_literals, absolute_import, print_function, division
 
 from datetime import datetime
-import re
-
```

Comment:
Leave this blank line in; when we start enforcing more import style rules, those will insist that stdlib and third-party imports be separated by a blank line.

Reply:
Forgot `requests` isn't stdlib, thanks >.>